server/src/metrics/actix_web_metrics.rs

```diff
@@ -331,10 +324,7 @@ impl PrometheusMetricsHandler {
         let encoder = TextEncoder::new();
         let metric_families = self.registry.gather();
         let mut buf = Vec::new();
-        if let Err(err) = encoder.encode(&metric_families[..], &mut buf) {
-            global::handle_error(MetricsError::Other(err.to_string()));
-        }
-
+        let _ = encoder.encode(&metric_families[..], &mut buf);
```

Comment:
Are we missing here the global handle error call?

Reply:
I removed it because it wasn't being exposed anymore